build/terraform/template/aws/cluster.tf.tpl

```diff
@@ -1,9 +1,14 @@
 // Manage state file with terraform cloud
-// terraform {
-//        backend "remote" {
-//        hostname = "app.terraform.io"
-//    }
-//}
+terraform {
+    backend "remote" {
+        hostname = "app.terraform.io"
+        organization = "Sifchain"
+
+        workspaces {
+            name = "sifchain-aws-feature-testnets"
```

Comment:
Hey this should only be edited once the template has been scaffolded to `./live/*` Can we revert this.